### ISSUE TITLE
Improve writing quality

### ISSUE THREAD

**Issue by cooljeanius** [cooljeanius/Two_Brothers_With_A_Third_Difficulty#4]
mentioned here: https://forums.serenesforest.net/index.php?/topic/86633-parrhesia-and-integritys-guide-to-surviving-wesnoth/&tab=comments#comment-5403061 

Specific points:
- Make Nil-Galion's motivations more sensical; mentioned here: https://forums.serenesforest.net/index.php?/topic/86633-parrhesia-and-integritys-guide-to-surviving-wesnoth/&tab=comments#comment-5403484 
- Give "Guarded Castle" a better scenario name; mentioned here: https://forums.serenesforest.net/index.php?/topic/86633-parrhesia-and-integritys-guide-to-surviving-wesnoth/&tab=comments#comment-5404284

**Comment by cooljeanius** [comment]
Note: @knyghtmare agrees on the Nil-Galion point: 
<img width="936" alt="Screen Shot 2021-02-06 at 06 42 43" src="https://user-images.githubusercontent.com/1388660/107149940-5ffe2980-6929-11eb-92bb-fed7dad624f2.png">

**Comment by cooljeanius** [comment]
On Discord @brirush84 offered to help with this; note that this add-on is based on the version of AToTB from 1.14, so there might be some prose changes for AToTB for 1.16 that address this that I might want to backport...

**Comment by brirush84** [comment]
Looking at the main github, there are maybe 2 changes to the script in the last few years:
https://github.com/wesnoth/wesnoth/commits/f5ea3b0906ff5bb3f1282a33bc3419c11e1cb105/data/campaigns/Two_Brothers

What date is your fork from?

Also, I'm interested in knowing what 'level' of rewriting you'd like. Here's a sample from the first scenario:

"If I could but face this ‘Mordak’! I think my magic might prove stronger than his. But he bides in the hills, well-guarded by his servants, and I muster frightened peasants to fight his minions with blades and sticks. I need my brother; he always had a better head for battle than I."

There are 3 ways I could approach this:
1. Only making changes to the biggest things called out in that post. This paragraph was part of a whole group called 3'/10', but not mentioned individually, so it would be left alone.

2. Rewording to use non-archaic English (more like the other campaigns) and simplifying some of the language:

"If only I could find ‘Mordak’, I think my magic might be strong enough to defeat him! But he bides his time in the hills, well-guarded by his servants, while I lead frightened peasants against his minions with blades and sticks. I need my brother; he has a better head for battle than I ever did."

3. Rebuild each paragraph with new dialog that suits the same purposes but doesn't follow the same structure.
I led a patrol yesterday in the woods searching for this Mordak, but all we found were his undead servants. No one was lost this time, even though the townsfolk still aren't used to battle. I'm growing frustrated with my failures. I'm sure Arvith would be able to find Mordak; my brother has a talent for thinking like the enemy. 

Which version would you prefer? None of them would take too long, given the small size of the campaign.

**Comment by cooljeanius** [comment]
Hm, I'm not really too sure, actually... another goal I have is getting translators to update the translations, so I don't want to make their work TOO hard on them by making things too different, but then again, since I'm going the add-on route here instead of trying to get these changes integrated back upstream, making things more different would give this a greater reason to exist... I dunno, I'll have to think about it...

**Comment by cooljeanius** [comment]
Right, I guess this was the main commit I was thinking of when it came to mainline prose changes I'd want to backport: https://github.com/wesnoth/wesnoth/commit/d4739088da5a24fd4cc94d0677c4d5da35cac082#diff-2738c52cec146069e040fb3dbc3f49c9e31f4c1b4c9d176cc0cecdb271555bd9 
(this add-on was based off of the version of AToTB in 1.14.12 I think; initial commit is from September 29, 2020, at which point this would have been my latest update to my README about add-on version numbers in my general mods collection: https://github.com/cooljeanius/wesnoth_mods/commit/71535e763074399275f474d2fd7debfc84f662ca#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 (so either 1.14.12 or 1.14.14, then))

**Comment by cooljeanius** [comment]
> Right, I guess this was the main commit I was thinking of when it came to mainline prose changes I'd want to backport: [wesnoth/wesnoth@d473908#diff-2738c52cec146069e040fb3dbc3f49c9e31f4c1b4c9d176cc0cecdb271555bd9](https://github.com/wesnoth/wesnoth/commit/d4739088da5a24fd4cc94d0677c4d5da35cac082#diff-2738c52cec146069e040fb3dbc3f49c9e31f4c1b4c9d176cc0cecdb271555bd9)

That commit originally came from this PR, for reference: https://github.com/wesnoth/wesnoth/pull/5255

**Comment by cooljeanius** [comment]
Oh, there's also this commit to consider: https://github.com/wesnoth/wesnoth/commit/c1cbc533534b8b428a8088c0fc312bf4e85a38f4

**Comment by cooljeanius** [comment]
> Oh, there's also this commit to consider: [wesnoth/wesnoth@c1cbc53](https://github.com/wesnoth/wesnoth/commit/c1cbc533534b8b428a8088c0fc312bf4e85a38f4)

...and that comes from here: https://github.com/wesnoth/wesnoth/pull/5815

**Comment by brirush84** [comment]
I forked your repository and retooled the first two scenarios. My goals were to eliminate plot holes, remove incorrect or ugly use of archaic language, and unify characterization. 

This makes the text very simple and plain, so it may not be the effect you wanted. Here is the commit: https://github.com/brirush84/Two_Brothers_With_A_Third_Difficulty/commit/2bb3923a7a700d84f6287a02afb886aaa849f596

If you don't like it, I can try a different style or you can go with a different author; thought I'd just throw these out there though!

**Comment by brirush84** [comment]
Also, I haven't done the last few scenarios, because I wanted to see your opinion. I'm totally okay if it's not your style!

**Comment by cooljeanius** [comment]
Sorry, haven't had a chance to look at this yet... thanks, though! I guess since the ball's back in my court I'll switch the assignee to me...

**Comment by cooljeanius** [comment]
I think I'm going to have to postpone this for a post-1.16 release, actually; I want to get 1 final release out on the 1.14 server while that's still the latest stable version, and don't really have time to be reviewing/testing any major changes before 1.16...

**Comment by cooljeanius** [comment]
I think I'm also going to ask @max-torch for input on this, given all the copyediting he's been doing of mainline

**Comment by cooljeanius** [comment]
ok un-postponing this now that I've uploaded this to the 1.16 server

**Comment by cooljeanius** [comment]
Thread from the previous writing overhaul: https://forums.wesnoth.org/viewtopic.php?t=31502
